onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -6532,10 +6551,20 @@ def aten_view_copy(self: TensorType, size: INT64) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_vstack(tensors: Sequence[TensorType]) -> TensorType:
+@torch_op("aten::vstack")
+def aten_vstack(tensors: Sequence[TTensor]) -> TTensor:
     """vstack(Tensor[] tensors) -> Tensor"""
 
-    raise NotImplementedError()
+    # TODO: Support at_least2d
```

Comment:
If I added the code from #767 to here for atleast_2d. It pops error saying "onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : This is an invalid model. Error: Duplicate definition of name (_inline_SequenceMaprank)." in eager mode, and "onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: Invalid tensor data type 0." in graph test.

cc @gramalingam @justinchuby any idea?

Reply:
```python
@torch_op("aten::vstack")
def aten_vstack(tensors: Sequence[TTensor]) -> TTensor:
    """vstack(Tensor[] tensors) -> Tensor"""

    # TODO: This is exactly from aten_atleast_2d
    @graph()
    def reshape_to_2dd(tensor):
        shape = op.Shape(tensor)
        rank = op.Size(shape)
        if rank <= 1:
            tensor = op.Reshape(tensor, op.Constant(value_ints=[1, -1]))
        return tensor
    
    new_tensors = op.SequenceMap(tensors, body=reshape_to_2dd)

    return op.ConcatFromSequence(new_tensors, axis=0)
```

Reply:
In eager mode, each op will be executed independently as an ONNX graph. Could you start from checking which op caused the "invalid model" error?